### DISCUSSION TITLE
Webpack Dynamic Module Error Management Problem

### DISCUSSION THREAD

**Discussion by trixmaxzero** [webpack/webpack#17035] [Q&A] [answered]
Hello,

My Code:

```

function loadComponent(scope, module) {
     return async() => {
         await __webpack_init_sharing__("default");
         const container = window[scope];
         await container.init(__webpack_share_scopes__.default);
         const factory = await window[scope].get(module);
         const Module = factory();
         return Module;
     };
}

```

"Module "./AppModule" does not exist in container. "

I am getting this error code. I'm trying Try-Catch or a different control, it doesn't work, how can I solve it?

 ```
const Component = React.lazy( loadComponent(props.scope, props.module) );

         return (
                 <Suspense fallback={<LoadView status={"loading"} title={"Module Loading"}/>}>
                     <Component {...props?.props} />
                 </Suspense>
         );
```

**Comment by alexander-akait** [comment]
It means you don't have `AppModule` module, check your bundle and configuration

**Reply by trixmaxzero** [reply]
Yes, I know but i can't socution for check of the AppModule

**Reply by alexander-akait** [reply]
What you try to solve? Sorry I don't undestand

**Reply by trixmaxzero** [reply]
I can't check if AppModule exists. I tried but failed. How can I find a solution for this.

I want to check if AppModule exists or not.

**Reply by alexander-akait** [reply]
Please provide code how you try it

**Reply by trixmaxzero** [reply, marked as answer]
Thank you for your quick reply, 

i solved the problem with https://github.com/Weiot-Cloud/universe/tree/main/packages/utilities 

thank you again.
